### ISSUE TITLE
[Accessibilité] Listes - affichage des tableaux en responsive

### ISSUE THREAD

**Issue by hmeneuvrier** [MTES-MCT/histologe#5199]
- [ ] Supprimer la scrollbar horizontale sur les tableaux ? (je ne suis pas sûre qu'il faille le faire, je ne me rappelle plus sur Stop-Punaises, du coup je préfère le noter) :

<img width="605" height="712" alt="Image" src="https://github.com/user-attachments/assets/e8a14a9a-798a-4657-aff2-bf964bc3543a" />

https://www.systeme-de-design.gouv.fr/version-courante/fr/composants/tableau/accessibilite-du-tableau

https://access42.net/integrer-tableaux-html-accessibles-conformes-rgaa/

Sur Stop-Punaises on a supprimé les scrollbar verticales (en utilisant datatable apparemment) :

<img width="563" height="695" alt="Image" src="https://github.com/user-attachments/assets/dc013c93-59e8-44c5-9aea-3761864a1cd9" />

<img width="945" height="529" alt="Image" src="https://github.com/user-attachments/assets/3d46adfe-d68b-450d-b4b1-e1edfae1535e" />
